nengo_viz/components/component.py

```diff
@@ -4,6 +4,8 @@ class Component(object):
     def __init__(self, viz, config, uid):
         self.config = config
         self.uid = uid
+        self.new_component = None
```

Comment:
What is this for? I grepped and couldn't find any other references to `new_component`...